pkg/network/tracer/ebpf_conntracker.go

```diff
@@ -90,10 +90,6 @@ var ebpfConntrackerPrebuiltCreator func(*config.Config) (bytecode.AssetReader, [
 
 // NewEBPFConntracker creates a netlink.Conntracker that monitor conntrack NAT entries via eBPF
 func NewEBPFConntracker(cfg *config.Config) (netlink.Conntracker, error) {
-	if !cfg.EnableEbpfConntracker {
```

Comment:
Do we have tests that are testing for this?

Reply:
yes we have `TestConntrackerFallback`